tests/nested/manual/muinstaller-real/gadget-partial.yaml

```diff
@@ -0,0 +1,48 @@
+volumes:
+  pc:
+    partial: [schema, structure, filesystem, size]
```

Comment:
if we list schema here (which I'm not sure we must initially) then the spec mentions that type needs to be `<mbrtype>,<gptguid>`. It might be simpler for the initial example to leave schema out of here and use schema: gpt?